Makefile

```diff
@@ -80,7 +80,7 @@ dist: apply \
 deploy: dist
 ifneq ($(and $(MM_SERVICESETTINGS_SITEURL),$(MM_ADMIN_USERNAME),$(MM_ADMIN_PASSWORD),$(HTTP)),)
 	@echo "Installing plugin via API"
-		(TOKEN=`http --print h POST $(MM_SERVICESETTINGS_SITEURL)/api/v4/users/login login_id=$(MM_ADMIN_USERNAME) password=$(MM_ADMIN_PASSWORD) | grep Token | cut -f2 -d' '` && \
+		(TOKEN=`http --print h POST $(MM_SERVICESETTINGS_SITEURL)/api/v4/users/login login_id=$(MM_ADMIN_USERNAME) password=$(MM_ADMIN_PASSWORD) X-Requested-With:"XMLHttpRequest" | grep Token | cut -f2 -d' '` && \
```

Comment:
100% agree. I'm always hesitant to change functionality during dev mode (since it's too easy to code against that), but this probably deserves some mention on developers.mattermost.com. Note that it's specific to http (vs. curl), because the former acts like a browser and can preserve cookies across invocations.